gehomesdk/erd/values/hood/common_enums.py

```diff
@@ -0,0 +1,34 @@
+import enum
+from typing import NamedTuple, Optional
+
+@enum.unique
+class ErdHoodLightSetting(enum.Enum):
+    OFF = 0
+    DIM = 1
+    HIGH = 2
+
+    def boolify(self) -> Optional[bool]:
+        return self != ErdHoodLightStatus.OFF
+
+    def percentify(self) -> Optional[int]:
+        return int(self.name.title())*50
+
+    def stringify(self, **kwargs):
+        return self.name.replace("_"," ").title()
+
+@enum.unique
+class ErdHoodFanSetting(enum.Enum):
+    OFF    = 0
+    LOW    = 1
+    MEDIUM = 2
+    HIGH   = 3
+    BOOST  = 4
+
+    def boolify(self) -> Optional[bool]:
+        return self != ErdHoodLightStatus.OFF
+
+    def percentify(self) -> Optional[int]:
+        return int(self.name.title())*50
```

Comment:
Not sure where these will be used, but should it be self.value instead of self.name.title()?

High will be 150%, and boost will be 200%, is that intentional?

Reply:
I don't think they will be used. I started to define how it could work, but feel it will end up being better put in the ha_gehome module. The rest of the calling functions in which would use those are not implemented. I figured better understand what to do with it once I start reviewing the other module.

Reply:
I just committed a update and removed those functions.